pkg/sql/importer/import_type_resolver.go

```diff
@@ -52,7 +52,7 @@ func makeImportTypeResolver(typeDescs []*descpb.TypeDescriptor) importTypeResolv
 // Note that if a table happens to have multiple types with the same name (but
 // different schemas), this implementation will return a "feature unsupported"
 // error.
-func (i importTypeResolver) ResolveType(
+func (i ImportTypeResolver) ResolveType(
```

Comment:
I'm wondering if there's a better `TypeResolver` is should use?

Reply:
This type resolver is insufficient for the sql write path. Ideally the type resolver used by LDR can take in foreign types with mismatched OIDs and allow the sql path to just work. Currently the sql path writes panic because we hydrate the datums ingested using the source side UDTs, which have a different OID than on the destination side.